contrib/bash/s2i

```diff
@@ -103,7 +103,7 @@ __handle_flag()
     local flagname=${words[c]}
     # if the word contained an =
     if [[ ${words[c]} == *"="* ]]; then
-        flagname=${flagname%!=(MISSING)*} # strip everything after the =
+        flagname=${flagname%=*} # strip everything after the =
```

Comment:
Looks like the `Printf` had further consequences interpreting Bash code :)
I had not think that far!